packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -102,23 +102,23 @@ Inserts the logo image that has been configured for your application. You can ch
 If you want to just change the logo image for your custom error pages, include the URL to the image instead of the macro:
 
 Example:
+
 ```html
 <img alt="{{orgName}}" src="https://example.com//SomeOtherImage.png" class="org-logo">
 ```
 
-### <span v-pre>`{{{errorSummary}}}`</span>
```

Comment:
we need this as well I think as this is in our html template? 
https://github.com/okta/okta-core/blob/master/resources/src/main/resources/templates/custom_pages/error.template.html#L35

Reply:
This is a duplicate. We've already listed it on line 76.